src/lib/commands.js

```diff
@@ -0,0 +1,274 @@
+import { TCModel, GVL } from "@iabtcf/core";
+import config from "./config";
+import {
+	encodeVendorConsentData,
+	encodeConsentData
+} from "./cookie/cookie";
+import {
+	CMP_VERSION,
+	CMP_ID,
+	COOKIE_VERSION
+} from "./init";
+import {SECTION_DETAILS, SECTION_VENDORS} from "./store";
+import log from "./log";
+
+const arrayFrom = require('core-js/library/fn/array/from');
+const COOKIE_VERSION_V1 = 1;
+
+const createCommands = (store) => {
+
+	/**
+	 * Converts vector to consent object
+	 */
+	const vectorToObject = (set) => {
+		let obj = {};
+
+		set.forEach((hasConsent, id) => {
+			obj[id] = hasConsent;
+		});
+
+		return obj;
+	};
+
+	 /**
+	 * Build consent fields object v1 from data that has already been persisted.
+	 */
+	const getConsentFieldsObject = () => {
+		const {
+			persistedConsentData
+		} = store;
+
+		const {
+			created,
+			lastUpdated,
+			cmpId_: cmpId = store.tcModel.cmpId,
+			cmpVersion_: cmpVersion = 1,
+			consentScreen_: consentScreen = store.tcModel.consentScreen,
+			consentLanguage_: consentLanguage = store.tcModel.consentLanguage,
+			vendorListVersion_: vendorListVersion,
```

Comment:
The same here - this value should not be read from consent string v2 but always passed in parameter to `getConsentFieldsV1` command.

Reply:
done